WebContent/WEB-INF/views/dashboard.jsp

```diff
@@ -140,7 +133,9 @@ org.springframework.web.context.support.WebApplicationContextUtils"%>
    <div class="row" >
      <a href="booking">  
   <div class="column">
-      
+     
+    <form action="booking">

```

Comment:
Should remove the <form> tag and add something like this currently in the dev branch

```
<a href="booking">
	<div class="column">
		<div class="card" id="hi" style="width: 400px; height: 300px;">
			<p>
				<i class="fa fa-user"></i>
			</p>
			<h1>Book</h1>
			<p>Book A Room</p>
			<p>
				<span class="glyphicon glyphicon-book"
					style="font-size: 150px;"></span>
			</p>
		</div>
	</div>
</a>
```